### ISSUE TITLE
Elements must have sufficient color contrast (.k-pager-info)

### ISSUE THREAD

**Issue by sryeluri** [telerik/kendo-angular#2038]
Below accessibility issue that needs investigation.

Issue Details
Elements must have sufficient color contrast
color-contrast

Element Path
.k-pager-info

Snippet
<span class="k-pager-info k-label">1 - 10 of 14 items</span>

Expected behaviour:
- Element has insufficient color contrast of 2.32 (foreground color: #aaaaaa, background color: #ffffff, font size: 11.3pt, font weight: normal). Expected contrast ratio of 4.5:1

To reproduce
Use K4W to investigate the issue details

Environment
Chrome version 70.0.3538.77

**Comment by ipeshev** [comment]
@sryeluri which theme is used in this case ?

**Comment by tsvetomir** [comment]
Thank you for the feedback. I can confirm that the default theme colors do not cover the WCAG contrast requirements.

A dedicated "high contrast" color palette will be included to address this issue. I don't have a timeline for this feature yet. The suggested workaround for the moment is to customize the theme colors as described in [Customizing Themes](https://www.telerik.com/kendo-angular-ui/components/styling/custom-themes/).

Moving this issue to [kendo-themes](https://github.com/telerik/kendo-themes/).
